e2e/steps_file.js

```diff
@@ -25,19 +26,25 @@ function log(msg) {
 module.exports = function () {
   return actor({
     async signIn(user) {
-      await this.retryUntilExists(async () => {
-        this.amOnPage(baseUrl);
+      if (currentUser !== user) {
+        this.say(`Logging in as ${user.email}`);
```

Comment:
maybe we should do currentUser={} here, just in case that entire process fail in the middle?
user logout but did not manage to log in

Reply:
makes sense, especially as that may happen in the nightly build.